cli/azd/cmd/testdata/TestUsage-azd-auth-login.snap

```diff
@@ -21,6 +21,6 @@ Global Flags
         --debug      	: Enables debugging and diagnostics logging.
         --no-prompt  	: Accepts the default value instead of prompting, or it fails if there is no default.
 
-Find a bug? Want to let us know how we're doing? Fill out this brief survey: https://aka.ms/azure-dev/hats.
+Find a bug? Want to let us know how we're doing? Fill out this brief survey: ]8;;https://aka.ms/azure-dev/hatshttps://aka.ms/azure-dev/hats]8;;.
```

Comment:
This should not be showing up this way. The snapshot we're generating here should be avoiding any color or formatted output. Similarly, make sure that this is working well with `docgen.go`.

Reply:
For the progres_log, I created a component to wrap the output and remove the special chars at the end.
You can see it at: pkg/input/progress_log_test.go
In case it helps as inspiration